tests/wpunit/_bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+/**
+ * Clear the uploads folder
+ */
+$_wp_content_upload_dir = WP_CONTENT_DIR . '/uploads';
```

Comment:
maybe `define( 'UPLOADS', 'wp-content/testuploads' );` so that we don't disturb any existing uploads